recipe/meta.yaml

```diff
@@ -21,17 +21,17 @@ requirements:
     - pip
     - python >=3.6
     - setuptools
-    - numpy
+    - numpy >=1.16.0
```

Comment:
If you left numpy out of the host requirements, would it still work? Here you should have exclusively dependencies that are necessary to install the package, not to run it.

Reply:
You're probably right. I put it in because I built the file locally using conda to upload to my own channel, and it complained about NumPy not being set (at least with the NeXpy build). However, the builds this week didn't complain so it may be safer to leave it out. I have removed it now.